packages/js/src/dashboard/scores/components/scores.js

```diff
@@ -21,11 +21,18 @@ import { TermFilter } from "./term-filter";
  * @returns {URL} The URL to get scores.
  */
 const createScoresUrl = ( endpoint, contentType, term ) => {
+	const existingParams = new URL( endpoint ).searchParams;
+
 	const searchParams = new URLSearchParams( { contentType: contentType.name } );
 	if ( contentType.taxonomy?.name && term?.name ) {
 		searchParams.set( "taxonomy", contentType.taxonomy.name );
 		searchParams.set( "term", term.name );
 	}
+
+	if ( existingParams.size !== 0 ) {
+		return new URL( "?" + existingParams + "&" + searchParams, endpoint );
+	}
+
 	return new URL( "?" + searchParams, endpoint );
```

Comment:
I think you can just use the searchParams from the URL in this case?
Disclaimer: untested with plain permalink

```js
	const url = new URL( endpoint );
	url.searchParams.set( "contentType", contentType.name );
	if ( contentType.taxonomy?.name && term?.name ) {
		url.searchParams.set( "taxonomy", contentType.taxonomy.name );
		url.searchParams.set( "term", term.name );
	}

	return url;
```

And the same for below.